packages/auto-label/src/auto-label.ts

```diff
@@ -211,66 +240,175 @@ handler.autoDetectLabel = (
   )?.github_label;
 };
 
-const BACKFILL_LABEL = 'auto-label:backfill';
+handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
+  owner: string,
+  repo: string,
+  issueNumber: number,
+  context: any,
```

Comment:
Is this truly an `any`?  Anything more specific we can do here?  As an aside - let's please treat linter warnings as errors.  This is something I intend to fix in the next release of `gts`.